AI/OpenAI/README.md

```diff
@@ -0,0 +1,55 @@
+# OpenAI
+
+In this document, you will find how to install and run a simple OpenAI example.
+
+## What is OpenAI?
+
+See OpenAI [website](https://openai.com/) and
+[documentation](https://platform.openai.com/docs/introduction).
+
+## Security
+
+Please, ensure that you carefuly read [Harvard's guidelines](https://huit.harvard.edu/ai/guidelines) on AI.
+
+## Installation
+
+You can install OpenAI in a conda/mamba environment:
+
+```bash
+[jharvard@boslogin01 ~]$ salloc --partition test --time 01:00:00 --mem-per-cpu 4G -c 2
+[jharvard@holy8a24301 ~]$ module load python/3.10.12-fasrc01
+[jharvard@holy8a24301 ~]$ export PYTHONNOUSERSITE=yes
+[jharvard@holy8a24301 ~]$ mamba create -n openai_env openai
+```
+
+## Run OpenAI
+
+You will need to provide an OpenAI key. You can generate one from
+[https://platform.openai.com/api-keys](https://platform.openai.com/api-keys).
+
+```bash
+# Request an interactive job
+[jharvard@boslogin01 ~]$ salloc --partition test --time 01:00:00 --mem-per-cpu 4G -c 2
+
+# Source conda environment
+[jharvard@holy8a24301 ~]$ mamba activate openai_env
+
+# set OpenAI key
+[jharvard@holy8a24301 ~]$ export OPENAI_API_KEY='my_key'
```

Comment:
Would be good to indicate that 'my_key' needs to be replaced by the API key generated on OpenAI's website.